File: src/main/java/me/miki/shindo/management/event/impl/EventPreRenderChunk.java
package me.miki.shindo.management.event.impl;

import me.miki.shindo.management.event.Event;
import net.minecraft.client.renderer.chunk.RenderChunk;

public class EventPreRenderChunk extends Event {

    private final RenderChunk renderChunk;

    public EventPreRenderChunk(RenderChunk renderChunk) {
        this.renderChunk = renderChunk;
    }

    public RenderChunk getRenderChunk() {
        return renderChunk;
    }
}

File: src/main/kotlin/me/miki/shindo/management/event/impl/EventPreRenderChunk.kt
package me.miki.shindo.management.event.impl

import me.miki.shindo.management.event.Event
import net.minecraft.client.renderer.chunk.RenderChunk

class EventPreRenderChunk(
    private val renderChunk: RenderChunk,
) : Event() {
    fun getRenderChunk(): RenderChunk = renderChunk
}
